src/Carbon/Lang/so.php

```diff
@@ -10,6 +10,50 @@
  */
 
 /*
- * Unknown default region, use the first alphabetically.
+ * Author:
+ * - Abdifatah Abdilahi(@abdifatahz)
  */
-return require __DIR__.'/so_DJ.php';
+return [
+    'year' => '{1}:count sanad|{0}:count sanadood|]1,Inf[:count sanadood',
```

Comment:
This is the default pluralization for "so" from symfony/translation, it means you can write:
```php
'year' => ':count sanad|:count sanadood',
```
It output `sanad` for `1` and `sanadood` for any other number.